.github/workflows/create-prs.yml

```diff
@@ -0,0 +1,73 @@
+name: Create PRs
```

Comment:
A script that should help me recover when branch pushes succeeded but the PR creation step failed. To be debugged after the merge.